src/ui/public/paginated_table/paginated_table.js

```diff
@@ -25,16 +27,14 @@ uiModules
         direction: null
       };
 
-      self.sortColumn = function (colIndex) {
+      self.sortColumn = function (colIndex, sortDirection) {
         var col = $scope.columns[colIndex];
 
         if (!col) return;
         if (col.sortable === false) return;
 
-        let sortDirection;
-
         if (self.sort.columnIndex !== colIndex) {
-          sortDirection = 'asc';
+          sortDirection = sortDirection || 'asc';
```

Comment:
Can you get a test for the default case here (for when you do not explicitly set a `sortDirection`)?